src/main/java/duke/controllers/MainWindow.java

```diff
@@ -0,0 +1,70 @@
+package duke.controllers;
```

Comment:
I am not really sure of using 'controllers' in the package name. I know that in JavaFX they call it FXML controller. However, the FXML controller comes under View in MVC as it is in charge of handling events and handling the ui logic.